app/src/main/java/com/keylesspalace/tusky/db/AccountEntity.kt

```diff
@@ -0,0 +1,67 @@
+/* Copyright 2018 Conny Duck
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>. */
+
+package com.keylesspalace.tusky.db
+
+import android.arch.persistence.room.Entity
+import android.arch.persistence.room.Index
+import android.arch.persistence.room.PrimaryKey
+
+@Entity(indices = [Index(value = ["domain", "accountId"],
+        unique = true)])
+data class AccountEntity(@field:PrimaryKey(autoGenerate = true) var id: Long,
```

Comment:
Not important: your formatting does not follow guidelines. There should be a newline after opening paren and before closing one.